src/io/flutter/sdk/FlutterSettingsConfigurable.java

```diff
@@ -66,34 +64,11 @@ protected void textChanged(final DocumentEvent e) {
       }
     });
 
-    final TextComponentAccessor<JComboBox> textComponentAccessor = new TextComponentAccessor<JComboBox>() {
-      @Override
-      public String getText(final JComboBox component) {
-        return component.getEditor().getItem().toString();
-      }
-
-      @Override
-      public void setText(@NotNull final JComboBox component, @NotNull final String text) {
-        if (!text.isEmpty() && !FlutterSdkUtil.isFlutterSdkHome(text)) {
```

Comment:
Is this validation made unnecessary by the logic in `apply`?  (Is that why you're deleting it?)